### ISSUE TITLE
[Bug]: NON-COMBAT section in sidemenu disappears permanently when toggling it's visibility

### ISSUE THREAD

**Issue by adibarra** [MelvorIdle/melvoridle.github.io#2513]
### Describe the bug

When toggling sidebar visibility for the NON-COMBAT sub-menu, the entire sub-menu along with it's nav header (containing the visibility button) fully disappear. Additionally, the NON-COMBAT visibility button also toggles visibility of the COMBAT section. The game needs to be restarted/reloaded for the sub-menu to reappear. Short gif of bug is attached.

### Reproduction Steps

1. Click on NON-COMBAT visibility toggle button
2. Entire section disappears including header

### Expected behaviour

The NON-COMBAT visibility toggle button should not make the whole section disappear while hiding toggle button permanently until next restart/reload. The visibility toggle button for the COMBAT section behaves as expected.

### Save Link

Not save related. Occurs on a fresh save game.

### Screenshots

![8R8VniXD5I](https://user-images.githubusercontent.com/93070681/177249803-d946eb2a-56f1-463a-8bbe-91c02d8ed8a5.gif)

### Console output

No error output

### Which platforms are you experiencing this bug on?

- [X] Chrome
- [X] Firefox
- [X] Edge
- [X] Safari
- [X] Mobile App (iOS)
- [X] Mobile App (Android)
- [X] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.0.5

### Game Subversions

?2073 through ?2077

### Are you using any scripts or extensions?

No

**Comment by aldousWatts** [comment]
Can confirm this still exists in the game, Chrome w/ no extensions, ?2075

Using this in the console successfully toggles .d-none (visibility) for the section without also toggling for the header.

`$($(".nav-main-heading:contains('Non-Combat')")).nextUntil('[id=nav-main-heading]').toggleClass('d-none');`

this code is adapted & should probably replace the line in the following game function `main.js`

```js
function toggleMenu(menu) {
    ...
    } else if (menu === 1) { //this is what is called by the non-combat eye
        $($('.nav-main-heading')[4]).nextUntil('[id=nav-main-heading]').toggleClass('d-none'); //this line is the issue
        if (skillsMenu) {
            $('#skill-menu-icon').attr('class', 'far fa-eye-slash text-muted ml-1');
            skillsMenu = false;
        } else {
            $('#skill-menu-icon').attr('class', 'far fa-eye text-muted ml-1');
            skillsMenu = true;
        }
    }
}
```

**Comment by aldousWatts** [comment]
if you're savvy with userscripts and you want a fix now, here you go. it was driving me a bit crazy so all this does is overwrites the game's toggleMenu function with the line mentioned above. works for me

```js
// ==UserScript==
// @name         Fix Non-Combat Toggle
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  overwrite global game function with fixed
// @author       AldousWatts
// @match        https://melvoridle.com/*
// @icon         https://www.google.com/s2/favicons?sz=64&domain=melvoridle.com
// @grant        none
// ==/UserScript==

//straight flat global override
window.toggleMenu = (menu) => {
    let c = [6, 7, 8, 9, 12, 16, 17, 18];
    if (menu === 0) {
        for (let i = 0; i < c.length; i++) {
            $('#nav-skill-tooltip-' + c[i]).toggleClass('d-none');
        }
        if (combatMenu) {
            $('#skill-menu-icon-1').attr('class', 'far fa-eye-slash text-muted ml-1');
            combatMenu = false;
        } else {
            $('#skill-menu-icon-1').attr('class', 'far fa-eye text-muted ml-1');
            combatMenu = true;
        }
    } else if (menu === 1) {
        $($(".nav-main-heading:contains('Non-Combat')")).nextUntil('[id=nav-main-heading]').toggleClass('d-none'); //this line is a fix applied to the game
        if (skillsMenu) {
            $('#skill-menu-icon').attr('class', 'far fa-eye-slash text-muted ml-1');
            skillsMenu = false;
        } else {
            $('#skill-menu-icon').attr('class', 'far fa-eye text-muted ml-1');
            skillsMenu = true;
        }
    }
}
```

**Comment by MelvorIdle** [comment]
Fixed as of v1.1. Please open a new report if issue persists.
